pkg/services/accesscontrol/ossaccesscontrol/folder.go

```diff
@@ -98,14 +98,19 @@ func ProvideFolderPermissions(
 			ctx, span := tracer.Start(ctx, "accesscontrol.ossaccesscontrol.ProvideFolderPermissions.ResourceValidator")
 			defer span.End()
 
-			query := &dashboards.GetDashboardQuery{UID: resourceID, OrgID: orgID}
-			queryResult, err := dashboardStore.GetDashboard(ctx, query)
+			ident, err := identity.GetRequester(ctx)
 			if err != nil {
 				return err
 			}
 
-			if !queryResult.IsFolder {
-				return errors.New("not found")
+			_, err = folderService.Get(ctx, &folder.GetFolderQuery{
```

Comment:
@chaudyg I noticied that dashboard store was used here to verify the existance of a folder. This worked because we dual write them to the legacy sql store but would have stop working as soon as we stop doing that